gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/components/QETopBar.kt

```diff
@@ -24,7 +26,9 @@ import com.gravatar.quickeditor.R
 import com.gravatar.ui.GravatarTheme
 
 @Composable
-internal fun QETopBar(onDoneClick: () -> Unit, modifier: Modifier = Modifier) {
+internal fun QETopBar(onDoneClick: () -> Unit, gravatarIconUrl: String, modifier: Modifier = Modifier) {
```

Comment:
```suggestion
internal fun QETopBar(onDoneClick: () -> Unit, onGravatarIconClick: () -> Unit = {}, modifier: Modifier = Modifier) {
```

I would expose the `onGravatarIconClick` rather than pass the URL to let the parent composable decide what behavior they want on icon tap. I think it's more flexible, WDYT?

Reply:
Fixed in 187258d